tools/scripts/upload-ab-to-s3.js

```diff
@@ -98,27 +98,28 @@ function getIdFromUrl (url) {
 }
 
 (async function () {
-  const filePaths = [
-    ...(env !== 'dev' ? [current] : []), // defaults to current build
-    next, // defaults to dev build
-    ...((env === 'dev' && await getOpenPrNums()) || []).map(num => `https://js-agent.newrelic.com/pr/PR-${num}/nr-loader-spa.min.js`)
-  ]
-
-  Promise.all(filePaths.map(fp => getFile(fp))).then((contents) => {
-    contents = contents.filter(([url, res]) => res.statusCode === 200)
-    if (!contents.length) throw new Error('Contents are empty')
-
-    console.log(`found ${contents.length} valid PR builds in CDN`)
-    let output = `window.NREUM=${JSON.stringify(config)};`
-    output += `
+  connectToS3(role, dry).then(async () => {
+    const filePaths = [
+      next, // defaults to dev build
+      ...(env !== 'dev' ? [current] : []), // defaults to current build
+      ...((env === 'dev' && await getOpenPrNums()) || []).map(num => `https://js-agent.newrelic.com/pr/PR-${num}/nr-loader-spa.min.js`),
+      ...((env === 'staging' && await getExperiments()) || []).map(s3ObjectName => `https://js-agent.newrelic.com/${s3ObjectName}`)
```

Comment:
Should we do this for `dev` too?  Could get rid of the PR build system completely and treat them the same between dev and staging

Reply:
PR builds do have a place i guess, since they don't always work as expected.  We should probably make sure we treat "experiments" as something polished enough to be in staging